nixos/modules/services/x11/desktop-managers/gnome.md

```diff
@@ -132,6 +132,27 @@ Unfortunately, we lack a way for these to be managed in a completely declarative
 So you have to enable them manually with an Extensions application.
 It is possible to use a [GSettings override](#sec-gnome-gsettings-overrides) for this on `org.gnome.shell.enabled-extensions`, but that will only influence the default value.
 
+## Nautilus Extensions {#sec-gnome-nautilus-extensions}
+
+Nautilus extensions can add additional functionality to Nautilus, usually via the right-click menu. Some of these extensions are dynamically loaded from `$NAUTILUS_4_EXTENSIONS_DIR` environment variable, which is automatically set if [](#opt-services.gnome.core-utilities.enable) is true (enabled by default if GNOME is enabled).
+
+Many Python based extensions can be found on the wild, and some of which are even packaged in Nixpkgs, (for example: `nautilus-open-any-terminal`). To load these extensions, you'll have to add the package that loads Python extensions for Nautilus:
+
+```nix
+environment.systemPackages = with pkgs; [
+  gnome.nautilus-python
+];
+```
+
+In fact, any package that includes a python script at `/share/nautilus-python/extensions`, will provide a nautilus Python extension. Other examples include:
```

Comment:
And this feels like a package specific docs which belong to project-specific docs: https://gitlab.gnome.org/GNOME/nautilus-python#running-extensions